woocommerce/woocommerce
# [Accessibility] Update product photoswipe top bar button order

### Submission Review Guidelines:

- * I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- * I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- * I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
- * Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes #51927 .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Browse to a product page with a product image or images.
2. Open the product image modal.
3. In the top bar, the actionable buttons such as close should receive tab focus in the order they are displayed.

Test Accessibility in using Chrome dev tools.

1. Open Dev Tools
2. Select "Accessibility" under "Elements"
3. Enable "Enable full-page accessibility tree"
4. Reload the page
5. Find "Dialog" in the Accessibility Tree
6. Buttons should be in the correct order.

eg; 
<img width="296" alt="image" src="https://github.com/user-attachments/assets/48c2a457-b100-4296-ab13-5b72890c0260">

### Changelog entry

- * [ ] Automatically create a changelog entry from the details below.
- * [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>
<summary>Changelog Entry Details</summary>
#### Significance
#### Type
</details>
